lucene/suggest/src/java/org/apache/lucene/search/spell/DirectSpellChecker.java

```diff
@@ -198,6 +200,19 @@ public void setMinQueryLength(int minQueryLength) {
     this.minQueryLength = minQueryLength;
   }
 
+  /** Get the maximum length of a query term to return suggestions */
+  public int getMaxQueryLength() {
+    return maxQueryLength;
+  }
+
+  /** 
+   * Set the maximum length of a query term (default: 0, i.e. no maximum)
+   * to return suggestions. 
+   */
```

Comment:
Can you add an explanation here on why this would be useful, i.e. too long terms will eat lots of memory.

Reply:
I've added a second line - happy to take others' input here if there are better words to use!